didx-xyz/yoma-mobile
# YOMA-591 - feat(MySkills): adding skill count to each of the user's skills

## Scope [YOMA-591](https://yomaworld.atlassian.net/browse/YOMA-591)

![](https://media.giphy.com/media/qsMnVPVX9WDI1obJWB/giphy.gif)
## Work Done
- added utils to calculate the number of times a user has attained a given skill
- adding 

## Steps to Test
- navigate to `MySkills` widget and click on the content to view all skills - see that there is a count beside each skill in both the view and the widget.
    
## Screenshots
![Screenshot 2022-05-26 at 23 29 18](https://user-images.githubusercontent.com/1424561/170583252-688f20a9-a240-43e8-9b16-6478e81df6d8.png)
![Screenshot 2022-05-26 at 23 29 30](https://user-images.githubusercontent.com/1424561/170583260-36b84b43-0a6b-4281-81fa-959d180fd88a.png)
